vllm/config.py

```diff
@@ -150,6 +150,14 @@ def __init__(
         self.hf_text_config = get_hf_text_config(self.hf_config)
         self.dtype = _get_and_verify_dtype(self.hf_text_config, dtype)
 
+        if getattr(self.hf_config, "max_position_embeddings", 0) == 131072:
```

Comment:
Add rope_scaling=None so that it doesn't affect other models that have rope scaling?

Reply:
good point, updated